lib/oli/publishing.ex

```diff
@@ -1235,6 +1236,28 @@ defmodule Oli.Publishing do
     Enum.map(results, fn [slug, title] -> %{slug: slug, title: title} end)
   end
 
+  def find_alternatives_group_references_in_pages(alternatives_resource_id, publication_id) do
```

Comment:
Using JSONB to directly query all resources for alternatives group references, same as objectives above